jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -549,12 +631,17 @@ public int getUsageCount()
         public String toString()
         {
             long encoded = state.get();
-            return String.format("%s@%x{usage=%d/%d,multiplex=%d/%d,pooled=%s}",
+            int hi = AtomicBiInteger.getHi(encoded);
+            int lo = AtomicBiInteger.getLo(encoded);
```

Comment:
I would rename these variables to the actual name of their value: `usageCount` and `multiplexingCount`.